src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,297 @@
+    ```
+
+    Example of a pipeline with split derivative directories:
+
+    ```Plain
+    <dataset>/derivatives/spm-preproc/sub-0001
+    <dataset>/derivatives/spm-stats/sub-0001
+    ```
+
+    Example of a pipeline with nested derivative directories:
+
+    ```Plain
+    <dataset>/derivatives/spm-preproc/sub-0001
+    <dataset>/derivatives/spm-preproc/derivatives/spm-stats/sub-0001
+    ```
+
+
+1.  As a standalone dataset independent of the source (raw or derived) BIDS
+    dataset. This way of specifying derivatives is particularly useful when the
+    source dataset is provided with read-only access, and for publishing
+    derivatives as independent bodies of work, or for describing derivatives
+    that were created from more than one source dataset. The `sourcedata/`
+    subdirectory MAY be used to include the source dataset(s) that were used
+    to generate the derivatives.
+    Likewise, any code used to generate the derivatives from the source data
+    MAY be included in the `code/` subdirectory.
+
+The rest of the Derivatives specification assumes Case 1, but Case 2 applies
+after removing `/derivatives/<pipeline_name>` from template names. In both cases
+every derivatives dataset is considered a BIDS dataset and must include a
+`dataset_description.json` file at the root level. Consequently, files should be
+organized to comply with BIDS to the full extent possible (i.e., unless
+explicitly contradicted below). Any subject-specific derivatives should be
+housed within each subject’s directory; if session-specific derivatives are
+generated, they should be deposited under a session subdirectory within the
+corresponding subject directory; and so on.
+
+## Derived dataset and pipeline description
+
+As for any BIDS dataset a `dataset_description.json` file MUST be found at the
+top level of the particular pipeline:
+`<dataset>/derivatives/<pipeline_name>/dataset_description.json`
+
+In addition to the keys for raw BIDS datasets,
+derived BIDS datasets include the following REQUIRED, RECOMMENDED or OPTIONAL
+`dataset_description.json` keys
+(a dot in the Key name denotes a key in a subdictionary):
+
+| **Key name**                  | **Description**                                                                                                                                                                                                                              |
+| ----------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| PipelineDescription.Name      | REQUIRED. Name of the pipeline that generated the outputs. In case the derived dataset is stored as a subfolder of the raw dataset this field MUST be a substring of the derived dataset folder name (a.k.a. `<pipeline_name>` - see above). |
```

Comment:
In #440, it was proposed to add `PipelineDescription` to `dataset_description.json` even for raw datasets, as these are also often produced by some tool, at least in part. The `Pipeline` terminology seemed less suitable to that case, so `ProducedBy` was suggested, and `GeneratedBy` was also suggested to better align with #440 (BEP 028 - Provenance). Further, because multiple tools might be used, the idea was that this `PipelineDescription` field could be a list.

In the interest of not cutting that discussion short, we should consider whether to keep this name and whether to permit or require this field to be a list. I would propose a quick straw poll by response emoji.

Name:

* :+1: PipelineDescription
* :smile: ProducedBy
* :tada: GeneratedBy

Numerosity
* :heart: One entry `{"Name": "X"}`
* :rocket: One or list `{"Name": "X"}` or `[{"Name": "X"}, {"Name": "Y"}]`
* :100: List `[{"Name": "X"}]` or `[{"Name": "X"}, {"Name": "Y"}]`

Tagging @bids-standard/derivatives @bids-standard/raw. Also BEP28 contributors @yarikoptic @cmaumet @satra @remiadon @jbpoline.

Feel free to write in other name suggestions.

Reply:
I cannot select a :100: reaction --> 
![image](https://user-images.githubusercontent.com/9084751/83064732-8feaf280-a062-11ea-8805-de8305c5dc7f.png)

is this normal?

Reply:
Sorry, I thought it was an option. Changed to eyes.

Reply:
There's a consensus forming on `GeneratedBy` and mandatory list (as opposed "object or list of objects"). So I'll go with that. Thanks for responding, folks!